fastlane/lanes/screenshots.rb

```diff
@@ -1,126 +1,86 @@
+# FIXME: Make this app dependent instead (WordPress vs Jetpack)
 RAW_SCREENSHOTS_DIR = File.join(Dir.pwd, "screenshots", "raw")
 RAW_SCREENSHOTS_PROCESSING_DIR = File.join(Dir.pwd, "screenshots", "raw_tmp")
 PROMO_SCREENSHOTS_PROCESSING_DIR = File.join(Dir.pwd, "screenshots", "promo_tmp")
-FINAL_METADATA_DIR = File.join(Dir.pwd, "metadata/android")
-
+FINAL_METADATA_DIR = File.join(Dir.pwd, "metadata", "android")
```

Comment:
Reminder to fix this before marking this PR as ready

Reply:
Note: This will be done as part of a subsequent PR after all (see `tooling/screenshots-lane-tweaks` branch) to not hold this one any longer.